src/handlers/addDependencyHandler.ts

```diff
@@ -19,6 +19,9 @@ export async function addDependencyHandler(options?: any): Promise<void> {
     } else if (options && options.projectBasePath) {
         // for "Maven dependencies" nodes from Project Manager
         pomPath = path.join(options.projectBasePath, "pom.xml");
+    } else if (options && options.project.pomPath) {
+        // for "Dependencies" node from module in Maven explorer
+        pomPath = options.project.pomPath;
     } else {
         // select a project(pomfile)
         const selectedProject: MavenProject | undefined = await selectProjectIfNecessary();

```

Comment:
when options = {}, now you don't enter this branch.

Reply:
when options = {}, you mean when the `DependenciesMenu` is empty? It needs further processed because now the `DependenciesMenu` is hidden if it is empty.

Reply:
That's a concrete example for NPE mentioned above. `addDependencyHandler` can be called elsewhere, e.g. from Project Manager, or directly from command palette etc.